### PR TITLE
basetest: Fix force_soft_fail being able to override the module status

### PR DESCRIPTION
**Pull request by okurz**
Verification run: http://lord.arch/tests/1944#step/force_soft_failure/6

### CODE REVIEW

File: doc/backend_vars.asciidoc [hunk 1]
```diff
@@ -1,36 +1,71 @@
 Supported variables per backend
 -------------------------------
 
+.AMT backend
```
Comment (coolo) [inline doc/backend_vars.asciidoc]: why?

Comment (okurz) [reply doc/backend_vars.asciidoc]: https://progress.opensuse.org/issues/34303 is the reason

Comment (okurz) [reply doc/backend_vars.asciidoc]: updated PR

**Comment by Martchus** [comment]
14-isotovideo.t and 99-full-stack.t are failing.

**Pull request closed without merging**